integration/shareddata/scalars.go

```diff
@@ -390,20 +396,22 @@ var Int64s = &Values[string]{
 		"int64-big-plus":  longBig + 1,
 		"int64-big-minus": longBig - 1,
 
-		// long representation of double max precision ~1<<53
-		"int64-prec-max":       int64(doubleMaxPrec),
-		"int64-prec-max-plus":  int64(doubleMaxPrec) + 1,
-		"int64-prec-max-minus": int64(doubleMaxPrec) - 1,
+		// long representation of double max precision ~1<<53 - 1
+		"int64-prec-max":          int64(doubleMaxPrec),
+		"int64-prec-max-plus":     int64(doubleMaxPrec) + 1,
+		"int64-prec-max-plus-two": int64(doubleMaxPrec) + 2,
+		"int64-prec-max-minus":    int64(doubleMaxPrec) - 1,
 
 		// negative long big values ~ -(1<<61)
 		"int64-neg-big":       -longBig,
-		"int64-neg-big-plus":  -(longBig + 1),
-		"int64-neg-big-minus": -(longBig - 1),
-
-		// long representation of double max precision ~ -(1<<53-1)
-		"int64-prec-min":       -int64(doubleMaxPrec),
-		"int64-prec-min-plus":  -(int64(doubleMaxPrec) - 1) + 1,
```

Comment:
This is missing, it should be `-int64(doubleMaxPrec)+2`

Reply:
Please take a look, I believe it's relevant https://github.com/FerretDB/FerretDB/pull/2324/files#r1152969755